Dependencies/xr/Include/XR.h

```diff
@@ -78,6 +86,17 @@ namespace xr
         bool IsValid{true};
     };
 
+    using NativePlaneIdentifier = void*;
```

Comment:
Is this ever dereferenced externally (by which I mean in NativeXr.cpp or beyond)? If not, should we really be exposing a pointer here?

Reply:
This is not dereferenced, but we do use this type as a key in the dictionary.  Is there a way I can expose this without declaring it explicitly as a void*?

Reply:
Spoke a little offline, but I think it might be valuable to go a little deeper here, so I sent out a meeting invite for Monday.